### PR TITLE
Fixed slider gradient for different value ranges

### PR DESCRIPTION
**Pull request by katyo**
The `value` incorrectly treated as percents for calculating slider gradients.
I fixed this by converting it from min..max range to 0..100.

### CODE REVIEW

File: src/components/Slider/Slider.svelte [hunk 1]
```diff
@@ -15,6 +15,9 @@
 
   export let classes = classesDefault;
 
+  $: percentFactor = 100.0 / (max - min);
+
+  $: toPercent = v => (v - min) * percentFactor;
```
Comment (matyunya) [inline src/components/Slider/Slider.svelte:20]: Why not put all calculations in the same reactive statement?

Comment (katyo) [reply src/components/Slider/Slider.svelte:20]: Hmm... sounds reasonable.
I'm quite newbie in svelte.
Will it cache calculation of factor?

Comment (matyunya) [reply src/components/Slider/Slider.svelte:20]: Both will change if `min` changes.
```
 $: toPercent = v => (v - min) * (100.0 / (max - min));
```
should work ok.

Comment (katyo) [reply src/components/Slider/Slider.svelte:20]: ok

**Comment by matyunya** [comment]
Thank you!

**Pull request merged**